### PR TITLE
Framework dropdown in the docs

### PR DESCRIPTION
**Pull request by wszymanski**
### Context

As the title says. Just a simple drop-down when version of docs is "frameworked".

Screenshot 2022-04-28 at 14 00 44

Screenshot 2022-04-28 at 14 00 39

[skip changelog]

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. #9310 

### Affected project(s):
- [ ] `handsontable`
- [ ] `@handsontable/angular`
- [ ] `@handsontable/react`
- [ ] `@handsontable/vue`
- [ ] `@handsontable/vue3`

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [x] My change requires a change to the documentation.

### CODE REVIEW

File: docs/.vuepress/theme/components/Frameworks.vue [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+const frameworkIdToFullName = new Map([
+  ['javascript', 'JavaScript'],
+  ['react', 'React'],
+  ['angular', 'Angular'],
+  ['vue2', 'Vue 2'],
+  ['vue3', 'Vue 3'],
+]);
+
+export default {
+  name: 'Frameworks',
```
Comment (jansiegel) [inline docs/.vuepress/theme/components/Frameworks.vue:23]: The `Frameworks` name for a component looks too generic to me. I know it's similar to the other dropdown, `Versions`, but I'm not a huge fan of that one either. Something like `FrameworkDropdown` or something along these lines would be more fitting, what do you think?

**Review by jansiegel — CHANGES_REQUESTED** [review]
The icon next to the framework name seems to be broken for the `next` version.
image

**Comment by wszymanski** [comment]
> The icon next to the framework name seems to be broken for the `next` version. image

Oh, I haven't mentioned that. But, it's already fixed within #9437.

**Comment by jansiegel** [comment]
> Oh, I haven't mentioned that. But, it's already fixed within #9437.

I'll take your word for it 👍

**Review by jansiegel — APPROVED** [review]

**Pull request merged**